scr/hw9/CountOccurrence.java

```diff
@@ -0,0 +1,49 @@
+package hw9;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class CountOccurrence {
+    public static void testCountOccurrence() {
```

Comment:
Similar should be absent in your classes. You may put it to main method. In the future we will add it into junit tests